modules/FollowMe.js

```diff
@@ -444,11 +450,15 @@ class FollowMe {
         if (smallVideo) {
             this.nextOnStageTimer = 0;
             clearTimeout(this.nextOnStageTimout);
-            /* eslint-disable no-mixed-operators */
-            if (pin && !VideoLayout.isPinned(clickId)
-                || !pin && VideoLayout.isPinned(clickId)) {
-                /* eslint-disable no-mixed-operators */
-                VideoLayout.handleVideoThumbClicked(clickId);
+
+            if (pin) {
```

Comment:
I know it's not part of the current change, just wondering why the double check here. 'pin' and 'clickId' seems to be redundant as clickId should be enough to decide if this event was a pin or unpin event. Or am I missing something?

Reply:
Brian had the same question. I think that's a smell that this code (like all code...) can be made better, or at least some things renamed or comments left.

clickId is the participant that got clicked. pin means whether or not to click, essentially. The double check is to prevent erroneous pinning and unpinning. So let's say User A has follow me enabled and clicks User B, the arguments User B receives here would be userBId and true. Let's say User B clicks User B to unpin User B. Then User A clicks User B to unpin User B; the arguments User B receives here would be userBId and false. If the double check were not here, User B might then pin User B again.